api/api.go

```diff
@@ -54,13 +55,14 @@ const (
 // API supports api requests to the cts biniary
 type API struct {
 	store   *event.Store
+	drivers map[string]driver.Driver
```

Comment:
Maps are by reference, and so it now seems that there are two objects accessing the same map of drivers. Albeit both are only reads (API and Controller), it does pose a potential area where one object could unintentionally modify the map.

^ the above callout isn't too concerning now, but an option could be to upgrade the map to a full on object to obscure the map because `driver.Drivers` is shorter to type than `map[string]driver.Driver`  😝

Reply:
Yes! Thanks for calling this out. I was considering a driver.Drivers since I think, to your point about currently there is only reads on the map, creating and deleting tasks will need to write. I chose the map so that I could focus on disable/enable tasks but hadn't considered the potential for unintentionally modifying the map. Making this change will definitely be first on the list for supporting create/delete taks cli.

Reply:
With create/delete on the road _map_ `driver.Drivers` makes more sense, good to wait for then for those changes

Reply:
Just following up here, this is now done (https://github.com/hashicorp/consul-terraform-sync/pull/218)! Thanks for the feedback!